### PR TITLE
Remove area edge lines in average PSD plots

### LINKED ISSUE #7738: raw.plot_psd: edge line of area plot
when plotting raw psd with `n_fft=len(raw.times), average=True` to get high def in frequency, the edge of the grey shaded area overplots, which creates an ambiguous darker grey area around the normal grey area
image
Here's a zoomed in version:
image

would be nice to be able to disable this edge line in the call to `raw.plot_psd`

### PR DESCRIPTION
**Pull request by hoechenberger**
#### Reference issue
Closes #7738

#### What does this implement/fix?
Do not plot lines on the edges of PSD areas

### CODE REVIEW

File: tutorials/raw/plot_40_visualize_raw.py [hunk 1]
```diff
@@ -192,3 +192,4 @@
 # .. LINKS
 #
 # .. _spectral density: https://en.wikipedia.org/wiki/Spectral_density
+
```
Comment (drammock) [inline tutorials/raw/plot_40_visualize_raw.py:195]: you'll need to revert this before we can merge; it will fail flake8

Comment (hoechenberger) [reply tutorials/raw/plot_40_visualize_raw.py:195]: Yep, just wanted to ensure the tutorial would be rendered… or would that have happened anyway?

Comment (hoechenberger) [reply tutorials/raw/plot_40_visualize_raw.py:195]: Oh I see, it doesn't render anything now haha

Comment (drammock) [reply tutorials/raw/plot_40_visualize_raw.py]: yeah, the triggering is determined by comparing the file against master.

What I should have said is "when you touch a tutorial to trigger it to render, a safe way to touch it is to just add/change a line break in a paragraph of text.  Adding blank lines in the middle of the document (e.g., in between paragraphs or between a paragraph and a code block) should also be safe.  Blank lines at the end make flake angry, and `XXX` comments will show up in the rendered tutorial (and so are also not a good choice).

**Comment by hoechenberger** [comment]
@agramfort @larsoner @dnacombo 
WDYT?

https://19932-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/raw/plot_40_visualize_raw.html#plotting-spectral-density-of-continuous-data

**Comment by larsoner** [comment]
Agreed

File: tutorials/raw/plot_40_visualize_raw.py [hunk 2]
```diff
@@ -188,6 +188,7 @@
 
 raw.plot_projs_topomap(colorbar=True)
 
+# XXX
```
Comment (agramfort) [inline tutorials/raw/plot_40_visualize_raw.py:191]:
```suggestion
```

**Comment by larsoner** [comment]
Thanks @hoechenberger

File: tutorials/raw/plot_40_visualize_raw.py (hunk 1 above)
Comment (hoechenberger) [reply tutorials/raw/plot_40_visualize_raw.py]: Thanks for the explanation, @drammock!

**Pull request merged**